### ISSUE TITLE
Add a rocks spec

### ISSUE THREAD

**Issue by aktau** [rgertenbach/Lua-Stats#3]
So this library can be installed with `luarocks`. Would be nice, I think.

**Comment by rgertenbach** [comment]
From a different similar request:

tl;dr I'd recommend using a different library such as SciLua.

In general I wouldn't really recommend using it, for a number of reasons:

I used this in a project by copy and pasting it, it's my first and last Lua project.

- It's not tested beyond close-enough-checks before committing, I didn't expect anyone to ever see this.
- It's slow, e.g. I don't see a reason for factorial to be recursive for integers when it can be so slow and an alternative is programmed, the integration is stepwise instead of using more sophisticated method like gradients
- That also tends to make it less exact for a given runtime
- I absolutely do not mind if you make your own version using any subset or all of this code but as mentioned cannot guarantee the usefulness or correctness.
